build-system/tsconfig.json

```diff
@@ -29,8 +29,8 @@
     "./tasks/coverage-map/*",
     "./tasks/create-golden-css/*",
     "./tasks/css/*",
-    // "./tasks/e2e/*", // 253 remaining type errors.
-    "./tasks/make-extension/*",
+    "./tasks/e2e/*", // 253 remaining type errors.
+    "./tasks/extension-generator/*",
```

Comment:
Why this change? (`extension-generator` no longer exists.)

Reply:
Good catch, I suppose this is the issue with resurrecting old PRs.
![image](https://user-images.githubusercontent.com/78179109/114627148-a0e82680-9c69-11eb-85c7-fdb41100d3b3.png)